### DISCUSSION TITLE
Target Featured Post inside CSS grid

### DISCUSSION THREAD

**Discussion by marvthedev** [strapi/strapi#7653] [Legacy Topics]
https://codepen.io/devvelox/pen/dyMzbNW

In my code pen (link above), I have 2 featured posts in specific spots on the css grid and the rest are just regular article posts. How would I place v-for just for the regular article posts without affecting the featured posts? How do I go about targeting the featured post on the grid if I created a boolean called "Featured" on my articles on strapi?

**Comment by derrickmehaffy** [comment]
Hello @mlmarinas we don't currently support questions related to frameworks outside of Strapi itself, I would recommend reaching out to the community on our slack https://slack.strapi.io 

We do plan to start a community forums that will allow asking questions like this, but Strapi ourselves can't provide much help in this area, surely if anyone in the community finds this question and can help you then great :smiley:
